tests/service/data/SyntaxTree/Expression/AnonymousRecords.fs

```diff
@@ -0,0 +1,6 @@
+{| X = 1 |}
```

Comment:
@auduchinok, I'm not covering all the grammar rules right here.
Any thoughts on how to reach `| LBRACE_BAR recdExprCore recover ` and `| LBRACE_BAR recover`?

Reply:
I guess something with an unfinished record at EOF would be the simplest way:

```fsharp
{| A = 1
```

```fsharp
{|
```

Reply:
Thanks, that worked.

Reply:
I thought about it a bit more and found more tips that could be useful in similar cases.

Sometimes it's also interesting to add another node afterwards and to see that recovered range doesn't include it and that the second node is parsed correctly:

```fsharp
{| A = 1
()
```

```fsharp
{|
()
```

Or even as part of a single expression:
```fsharp
do
    {|
    ()
```